### ISSUE TITLE
Deleteing the First QxtGlobalShortcut invalidates the hanlder and makes all other GlobalShortcuts stop working in Qt5

### ISSUE THREAD

**Issue by mrbesen** [MOZGIII/QxtGlobalShortcut#1]
When you have more than one Button, like this:
```
QxtGlobalShortcut* up = new QxtGlobalShortcut(QKeySequence("Shift+Up"));
QxtGlobalShortcut* down = new QxtGlobalShortcut(QKeySequence("Shift+Down"));

// both up and down work here

delete up;

//none of them work now
```

This is a problem in the way the nativeEventFilter is registered in Qt5. 
Making it a standalone instace should help.

**Comment by MOZGIII** [comment]
Hey, thx, feel free to submit a PR!

**Comment by mrbesen** [comment]
Allright, should be fixed by #2
